src/styles/box.scss

```
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform 0.3s ease;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
+  }
+
+  .card__title {
+    margin-top: 40px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+  }
+
+  .card__code {
```

why don't u use nesting here?
```suggestion
  .&__title {
    margin-top: 40px;
    margin-bottom: 4px;
    font-weight: 500;
    font-size: 12px;
  }

  .&__code {
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**